.github/workflows/main.yaml

```diff
@@ -22,8 +22,9 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: 1.68
```

Comment:
Can you set up a reminder in slack somewhere so we don’t forget to put this back in a while?

Reply:
Yes, will do that. This is really just a bandaid and I kinda hate it